carsFilter/src/main/java/tkhal.filter.controller/CarsFilter.java

```diff
@@ -0,0 +1,20 @@
+package tkhal.filter.controller;
+
+import model.Car;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CarsFilter {
+
+    private final static Logger LOGGER = LoggerFactory.getLogger(MessageFilterController.class);
+
+    public Car filter(Car tempCar) {
```

Comment:
I would suggest to return `boolean` instead for `Car`

Reply:
done